src/Performance/Listener/PerformanceLoggerSubscriber.php

```diff
@@ -54,7 +54,7 @@ final class PerformanceLoggerSubscriber implements EventSubscriber
     private $memoryFormatter;
 
     public function __construct(
-        Timer $timer,
+        Stopwatch $timer,
```

Comment:
What do you think about being consistent here: `Stopwatch` class and `$stopwatch` parameters, variables, properties?

Here and in other classes below.

(similar to what Symfony does https://symfony.com/doc/current/components/stopwatch.html#sections)

Reply:
totally, should be corrected